lnwire/features_test.go

```diff
@@ -396,3 +396,103 @@ func TestIsEmptyFeatureVector(t *testing.T) {
 	fv.Unset(StaticRemoteKeyOptional)
 	require.True(t, fv.IsEmpty())
 }
+
+// TestValidatePairs tests that feature vectors can only set the required or
+// optional feature bit in a pair, not both.
+func TestValidatePairs(t *testing.T) {
+	t.Parallel()
+
+	rfv := NewRawFeatureVector(
```

Comment:
Not sure about this one tbh, as it all validatoin lives in the feature manager rather than `RawFeatureVector` but that may be by coincidence rather than design.  cc: @positiveblue, any strong feelings here?

Reply:
I am fine with both